src/lib/pipeline_configuration.py

```diff
@@ -365,26 +365,32 @@ def validate(self):
 
 
 class RapidProKeyRemapping(object):
-    def __init__(self, rapid_pro_key, pipeline_key):
+    def __init__(self, activation_message, rapid_pro_key, pipeline_key):
```

Comment:
What is an activation message? Could we put this in the glossary?

Reply:
I wonder if I have named this right. On second reading it looks like it should contain a message rather than a bool...

Reply:
`is_activation_message`?

Reply:
+1, thanks

Reply:
changed